spec/system/admin/payments_stripe_spec.rb

```diff
@@ -192,6 +192,10 @@
         order.payments << payment
       end
 
+      after do
+        Stripe::Account.delete(connected_account.id)
+      end
```

Comment:
Just an idea to scale this further. In the spec helper we could override `Stripe::Account.create` to call the original but store all generated account ids in an array. Then after every test case we go through that list (mostly empty, but sometimes not) and call delete on every id to clean up. Then individual specs don't have to repeat this code.

Reply:
I think that's a great idea! It would be worth doing if it is likely that we'll add more specs with this in the future.